db-build.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+$servername = "localhost";
+$username = "root";
+$password = "";
+
+try {
+    $conn = new PDO("mysql:host=192.168.20.20", $username, $password);
```

Comment:
Database connection will be handled in a seperate file.

Reply:
I think Liz is creating those